core/src/server/Config.cpp

```diff
@@ -974,9 +975,12 @@ Config::CheckCacheConfigInsertBufferSize(const std::string& value) {
             return Status(SERVER_INVALID_ARGUMENT, msg);
         }
 
+        int64_t cache_size;
```

Comment:
variable need to be initialized a value.

Reply:
done